### PR TITLE
Eslint and other minor fix

### LINKED ISSUE #24: Eslint and other minor fixes

### PR DESCRIPTION
**Pull request by mrxshinji**
Fixes #24 

1. Configured eslint to be less aggressive
2. fixed import for ButtonStory and re-lint the rest

### CODE REVIEW

File: src/components/notifications/notifications.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    {
+      type: 'unusual',
+    },
+    {
+      type: 'receive',
+      from: 'Karios',
+      amount: 1,
+    },
+  ];
+  console.log(notificationData);
```
Comment (aftabrehan) [inline src/components/notifications/notifications.jsx:22]: Please, make sure to remove unused comments.

**Review by aftabrehan — CHANGES_REQUESTED** [review]
Excellent PR, @mrxshinji 💪 🚀 💯 

Just needs a minor fix and then it's good to go 🚀

**Comment by mrxshinji** [comment]
done changes

**Review by aftabrehan — APPROVED** [review]
Excellent PR, @mrxshinji 💪 🚀 💯

**Pull request merged**